### ISSUE TITLE
ImageObject.dotScreen() not applying to shadow()

### ISSUE THREAD

**Issue by chrisjansky** [typemytype/drawbot#547]
Hello,

Combining `ImageObject.dotScreen()` with `shadow()` (code below) produces the following result:

```
foo = ImageObject()

with foo:
    size(800, 800)
    fill(0.5)
    shadow((0, 0), 50)
    rect(50, 50, 700, 700)

foo.dotScreen()

image(foo, (100, 100))
```

<img width="793" alt="Screenshot 2023-11-21 at 9 15 51" src="https://github.com/typemytype/drawbot/assets/5342612/e2b38f58-bd2c-4d86-8e16-a3d9074b9211">

Is there any way for the blurred `shadow()` to be affected by `ImageObject.dotScreen()`? I’ve tried combining `ImageObject.dotScreen()` with `ImageObject.gaussianBlur()`, but got the same result.

Basically I’m looking for something like this:

<img width="627" alt="Screenshot 2023-11-21 at 9 29 14" src="https://github.com/typemytype/drawbot/assets/5342612/3d5d3db1-d4e1-410d-9844-afc0da2fb6ca">

Thanks!

**Comment by typemytype** [comment]
you need a solid background, no transparent colors:


```python
foo = ImageObject()

with foo:
    size(800, 800)
    # draw white background rect
    fill(1)    
    rect(0, 0, 800, 800)
    fill(0.5)
    shadow((0, 0), 50, (0, 0, 0))
    rect(50, 50, 700, 700)

foo.dotScreen()

image(foo, (100, 100))
```

**Comment by chrisjansky** [comment]
@typemytype Yes, that works, thank you!
